sonar-project.properties

```
@@ -20,15 +20,15 @@ sonar.eslint.reportPaths=packages/analytics-js/reports/eslint.json,packages/anal
 # Path to sources
 sonar.sources=packages
 sonar.inclusions=**/src/**/*.js, **/src/**/*.ts
-sonar.exclusions=**/*.json,**/*.html,**/*.png,**/*.jpg,**/*.gif,**/*.svg,**/nativeSdkLoader.js,assets/**/*
+sonar.exclusions=**/*.json,**/*.html,**/*.png,**/*.jpg,**/*.gif,**/*.svg,**/nativeSdkLoader.js,assets/**/*,**/event
```

Just `event` is too generic and may result in false positives.

```suggestion
sonar.exclusions=**/*.json,**/*.html,**/*.png,**/*.jpg,**/*.gif,**/*.svg,**/nativeSdkLoader.js,assets/**/*,packages/analytics-js-plugins/errorReporting/event/**/*
```

---

Addressed